src/about.html

```diff
@@ -39,186 +39,207 @@
 						<span class="icon-bar"></span>
 						<span class="icon-bar"></span>
 					</button>
-					<a class="brand" href="index.html"><img src="img/tosdr-icon-16.png" alt="" style="vertical-align:baseline;margin-right:.5em;" class="logo" />Terms of Service; Didn't&nbsp;Read</a>
+					<a class="brand" href="index.html"><img src="img/tosdr-icon-16.png" alt="Tos;DR Logo." style="vertical-align:baseline;margin-right:.5em;" class="logo" />Terms of Service; Didn't&nbsp;Read</a>
 					<div class="nav-collapse collapse">
 						<ul class="nav">
-							<li class=""><a href="index.html#services">Ratings</a></li>
+							<li class=""><a href="index.html#services" target="_blank">Ratings</a></li>
```

Comment:
```suggestion
							<li class=""><a href="index.html#services">Ratings</a></li>
```
No need for blanks on internal links